Payments

ericdesj opened the pull request:
This task was more annoying than anticipated. I tried three different libraries for credit card payments which for one reason or another didn't work (project version, know issues with the library, etc.). I created a simple payment form which upon entering payment information it will increment the tutor's revenue.

**Tasks completed**
- Created a view for the credit card payments
- Added business logic to validate all three payment fields
- Created a new endpoint in Tutors `incrementTutorRevenue(int tutorID, double revenue)` which increments the revenue of given tutor. I also created a new column for revenue which is set by default to `0.00`.

The payments page it an _in between_ task. Were missing the session sign up data to pass in the bundle to process a payment. Also, there's no point to anchor the payment fragment, so during development I set to the home view.

**To do**
- Pass `tutorID`, `tutoringFee` and `studentID` in the bundle
- Set the tutoring fee for the TextView
- Change temporary values in `incrementTutorRevenue()` method
- Determine what is the next intent - home view maybe?


![payment](https://cloud.githubusercontent.com/assets/6282512/24073297/3fe71bfe-0bd4-11e7-965c-8078c13769f3.gif)

Updated table (clicked on the increment twice)
![screen shot 2017-03-18 at 11 59 04 am](https://cloud.githubusercontent.com/assets/6282512/24073298/43c6193c-0bd4-11e7-99c4-2bcb896c21f7.png)

Mons1Oerjan commented on app/src/main/java/cs/dal/krush/LoginMainActivity.java:
```diff
@@ -36,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
         final EditText password = (EditText) findViewById(R.id.password);
         final Button login_button = (Button) findViewById(R.id.submitLogin);
-        final TextView krush_logo_textView = (TextView) findViewById(R.id.krushLogo);
+        final TextView krush_logo_textView = (TextView) findViewById(R.id.paymentHeader);
```
any reason as to why this has been changed from krushLogo to paymentHeader?

ericdesj replied:
Fixed this

Mons1Oerjan commented on app/src/main/java/cs/dal/krush/studentFragments/StudentPaymentFragment.java:
```diff
@@ -0,0 +1,100 @@
+import cs.dal.krush.R;
+import cs.dal.krush.models.DBHelper;
+
+public class StudentPaymentFragment extends Fragment {
```
add javadoc

ericdesj replied:
Done

ericdesj commented:
Added the credit card type validation:
![payment](https://cloud.githubusercontent.com/assets/6282512/24074357/c545611e-0be5-11e7-9813-00d7b1e98f4e.gif)

Also added the citations for the task.
